conanfile.py

```diff
@@ -6,20 +6,18 @@ class DummyConan(ConanFile):
     version = "1.0"
     topics = ("cmake")
     generators = "cmake_find_package"
-    exports_sources = ["CMakeLists.txt", "dummy-config.cmake", "my-super-cmake-file.cmake"]
+    exports_sources = ["CMakeLists.txt"]
+    settings = "os", "arch", "compiler", "build_type"
 
-    def configure_cmake(self):
+    def _configure_cmake(self):
         cmake = CMake(self)
         cmake.configure()
-
         return cmake
 
     def build(self):
-        cmake = self.configure_cmake()
+        cmake = self._configure_cmake()
         cmake.build()
 
     def package(self):
-        cmake = self.configure_cmake()
-        cmake.install()
-
-
+        cmake = self._configure_cmake()
+        #cmake.install()
```

Comment:
I have commented this as now there is nothing to install (no cpp files, no binaries generated)